samples/Calling/src/app/App.tsx

```diff
@@ -204,18 +204,16 @@ const App = (): JSX.Element => {
         return <Spinner label={'Getting user credentials from server'} ariaLive="assertive" labelPosition="top" />;
       }
       return (
-        <React.StrictMode>
```

Comment:
keep strict mode?

Reply:
@JamesBurnside, @emlynmac it is enabled in `index.tsx` for all Sample apps. The one that is deleted in this PR, is additional call in `App.tsx` file which is not needed anymore.

Reply:
Ah! 💖 the extra cleanup!